feat(payment): PAYMENTS-5443 Update docs to include human verification… (#892)
bigcommerce/checkout-sdk-js: README.md

#### Submit order

> And then, you can ask the customer to provide payment details required by their chosen payment method. If the method is executed successfully, you will create an order and thereby complete the checkout process. If human verification is required before completing the order (either automatically detected or by having the feature on), this wil be done in this step during submitting the payment.

Reviewer: How about something like
```suggestion
And then, you can ask the customer to provide payment details required by their chosen payment method. If the method is executed successfully, you will create an order and thereby complete the checkout process.

We may require human verification to be completed before payment can be processed, which will be handled during this step.
```
